### ISSUE TITLE
Create CheckboxButton.js component

### ISSUE THREAD

**Issue by tesla809** [COVID-19-electronic-health-system/Corona-tracker#83]
**Task**
Create **CheckboxButton.js** component.
See the following for the [design.](https://github.com/COVID-19-electronic-health-system/Corona-tracker/blob/master/design/wireframes/CORONATRACKER%20UI%20-%20Symptom%20Screen.png)

It is the checkbox in the view.

**What done looks like:**
1) You created the CheckboxButton.js component as per the instruction above.
2) Add comments to explain code if necessary.
3) We have CheckboxButton.js component in /components.
4) You have used the themePalette.css file and the corresponding classes to style the button and imported at the top of the file BEFORE your component.
5) Create CheckboxButton.css file. Use themePalette.css for base styling. It has the theme's colors.
6) You have used the name CheckboxButton.js for the component.

**Design considerations:**
We have decided to use regular CSS files to keep things simple for now. This allows for more contributors and less time to upskill to contribute.

Once MVP is done the team can discuss if we want to transition it to styled-components or something like CSS modules.

**Folder paths**
CSS is located in /client/src/css
Components are located in /client/src/components

**Style guide for CSS files:**
**Naming convention:**
Component -> CheckboxButton.js
CSS file -> CheckboxButton.css

**To avoid naming collisions:**
prefix class names with the name of component and link with kebab case.
Class name -> .CheckboxButton-header { ... }

**How to handle state:**
If you can use redux, use it.
Create action and reducer with an appropriate name.

**Testing:**
No need to worry about testing for now.
If you choose to add tests, please place in client/src/tests

**Comment by janetmndz** [comment]
@tesla809 May I take a crack at this one?

**Comment by tesla809** [comment]
Yup! have at it!

**Comment by friedger** [comment]
Is this the updated design file? https://github.com/COVID-19-electronic-health-system/Corona-tracker/blob/master/design/wireframes/FINAL%20V1%20-%2003.20/CORONATRACKER%20UI%20-%20Symptom%20Screen.png

**Comment by liadove** [comment]
I guess it is. But for this particular issue: https://raw.githubusercontent.com/COVID-19-electronic-health-system/Corona-tracker/master/design/wireframes/v1-final-MVP-subcomponents/survey-view-check-grouped-buttons-view.png

**Comment by liadove** [comment]
I almost finished functionality of buttons. But need to work on css

**Comment by AdhamAH** [comment]
@liadove are you still working on this?

**Comment by liadove** [comment]
@AdhamAH I thought we are going to talk about styling more clear, but the component itself already done. You can see it in the survey-view folder. If there's something I can work on I am free now, so I can pick an issue up.

**Comment by SomeMoosery** [comment]
Which PR closed this issue? Let's hold off on picking up new issues just for the time being while we finalize on a roadmap. We'll have a clearer vision on where we're going in just a little bit.

**Comment by liadove** [comment]
Good idea! This is the PR for the issue: https://github.com/COVID-19-electronic-health-system/Corona-tracker/pull/149
